### ISSUE TITLE
Undefined name 'ScaffoldMessenger'

### ISSUE THREAD

**Issue by moezzineb** [flutter/flutter#72248]
Always getting this error on a project without any reason for the moment could you help me on this.

Cheers

**Comment by iapicca** [comment]
@moezzineb 
can you provide your `flutter doctor -v` and a minimal reproducible code sample

**Comment by moezzineb** [comment]
Voila : 
https://imgur.com/zoJ1LvL
https://imgur.com/uWeMum8
https://imgur.com/oRQwVMq

hope it help you and if you need anything else just ask.

**Comment by adeolaex** [comment]
@moezzineb Those links weren't showing up the image(s).

Although I'll take a wild guess that you might not be on the `master channel` .
`ScaffoldMessenger `, is currently on the master channel as per Flutter's website 

> Landed in version: 1.23.0-13.0.pre In stable release: not yet

Try switching channels
```
flutter channel master
flutter clean
```

**Comment by pedromassangocode** [comment]
Hi @moezzineb 
This issue doesn't seem to describe a bug or a feature request. Please see https://flutter.dev/community for resources and asking questions like this,
you may also get some help if you post it on Stack Overflow and if you need help with your code, please see https://www.reddit.com/r/flutterhelp/
Closing, as this isn't an issue with Flutter itself. If you disagree, please write in the comments and I will reopen it.
Thank you

**Comment by moezzineb** [comment]
> 
> 
> @moezzineb Those links weren't showing up the image(s).
> 
> Although I'll take a wild guess that you might not be on the `master channel` .
> `ScaffoldMessenger `, is currently on the master channel as per Flutter's website
> 
> > Landed in version: 1.23.0-13.0.pre In stable release: not yet
> 
> Try switching channels
> 
> ```
> flutter channel master
> flutter clean
> ```

Just followed your steps and it works thank you very much.

**Comment by madhead** [comment]
I was trying to set up a CI for my Flutter project using this GitHub action: https://github.com/subosito/flutter-action. What I had to to is to use `beta` channel (`master` didn't work):

```yml
- uses: subosito/flutter-action@v1
  with:
    channel: beta
```
